date-input.html

```diff
@@ -9,8 +9,12 @@
 -->
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-input/iron-input.html">
-<link rel="import" href="../iron-flex-layout/classes/iron-flex-layout.html">
+<link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../iron-validatable-behavior/iron-validatable-behavior.html">
+<link rel="import" href="../paper-input/paper-input-container.html">
```

Comment:
Cool.. reminds me: the implementation of `--paper-input-container-font` et al are to be considered public implementation for the purposes of considering whether a change is breaking :skull: